hw1/src/test/java/com/epam/tc/hw2/ex1/Exercise1Test.java

```diff
@@ -0,0 +1,74 @@
+package com.epam.tc.hw2.ex1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.epam.tc.hw2.BaseTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Exercise1Test extends BaseTest {
+    private final String url = "https://jdi-testing.github.io/jdi-light/index.html";
+    private List<String> leftMenuList = Arrays
+            .asList("Home", "Contact form", "Service", "Metals & Colors", "Elements packs");
+    private List<String> textUnderIcons = Arrays.asList(
+            "To include good practices\nand ideas from successful\nEPAM project",
+            "To be flexible and\ncustomizable",
+            "To be multiplatform",
+            "Already have good base\n(about 20 internal and\nsome external projects),\nwish to get more…");
+
+    @Test(groups = {"hw2"})
+    public void ex1() {
+        //1. Open test site by URL
+        webDriver.navigate().to(url);
+        assertThat(webDriver.getCurrentUrl()).contains(url);
+        //2. Assert Browser title
+        assertThat(webDriver.getTitle()).isEqualTo("Home Page");
+        //3. Perform login
+        WebElement dropDown = webDriver.findElement(By.id("user-icon"));
+        dropDown.click();
+        WebElement login = webDriver.findElement(By.id("name"));
+        login.sendKeys("Roman");
+        WebElement password = webDriver.findElement(By.id("password"));
+        password.sendKeys("Jdi1234");
+        WebElement enter = webDriver.findElement(By.id("login-button"));
+        enter.click();
+        //4. Assert Username is loggined
+        WebElement userName = webDriver.findElement(By.id("user-name"));
+        assertThat(userName.isDisplayed()).isTrue();
+        assertThat(userName.getText()).isEqualToIgnoringCase("Roman Iovlev");
+        //5. Assert that there are 4 items on the header section are displayed and they have proper texts
+        List<WebElement> headerMenuElements = webDriver.findElements(By.xpath("/html/body/header/div/nav/ul[1]/li"));
+        for (int i = 0; i < headerMenuElements.size(); i++) {
+            assertThat(headerMenuElements.get(i).isDisplayed()).isTrue();
+            assertThat(headerMenuElements.get(i).getText()).isEqualToIgnoringCase(leftMenuList.get(i));
```

Comment:
lets rethink this assertion.
cus if something will be changed into menu (order as example) this asssertion will be failed